### PR TITLE
improve vagrant with kvm

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/improve_vagrant_with_kvm.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+Improve Vagrant with KVM
+========================
+
+Context
+-------
+
+A simple installation of `vagrant`_ will be working with `virtualbox`_. In some cases, you
+will notice that `virtualbox`_ is not the faster solution on the market. `KVM`_ can be a good
```
Comment (aloupfor) [inline en/developer_guide/improve_vagrant_with_kvm.rst:8]: the fastest solution?

File: en/developer_guide/improve_vagrant_with_kvm.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+The `libvirt`_ plugin should be added to vagrant
+
+.. code-block:: bash
+
+    $ vagrant plugin install vagrant-libvirt
+
+Launch the box
+~~~~~~~~~~~~~~
+
+Once those installation have been performed, if you have imported the right base box, you could launch
```
Comment (aloupfor) [inline en/developer_guide/improve_vagrant_with_kvm.rst]: those installations?

File: en/developer_guide/improve_vagrant_with_kvm.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+KVM installation
+~~~~~~~~~~~~~~~~
+
+You need to install all the packages for the vagrant plugin, and to allow `KVM`_ to run.
+
+.. code-block:: bash
+
+    $ aptitude install libxslt-dev libxml2-dev libvirt-dev zlib1g-dev qemu-utils libvirt-dev qemu-kvm
+    $ gem install ruby-libvirt -v '0.6.0'
```
Comment (aloupfor) [inline en/developer_guide/improve_vagrant_with_kvm.rst:22]: You may have to explain that you have to do (install) something in order to use `gem` command.

File: en/developer_guide/improve_vagrant_with_kvm.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+The `libvirt`_ plugin should be added to vagrant
+
+.. code-block:: bash
+
+    $ vagrant plugin install vagrant-libvirt
+
+Launch the box
+~~~~~~~~~~~~~~
+
+Once those installations have been performed, if you have imported the right base box, you could launch
```
Comment (ngilain) [inline en/developer_guide/improve_vagrant_with_kvm.rst:82]: you can instead of you could ?

File: en/developer_guide/improve_vagrant_with_kvm.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+Directly install the plugin
+
+.. code-block:: bash
+
+    $ vagrant plugin install vagrant-mutate
+
+Convert your box
+~~~~~~~~~~~~~~~~
+
+We suppose that you have already registered the ``bento/debian8.2`` base box.
```
Comment (ngilain) [inline en/developer_guide/improve_vagrant_with_kvm.rst:109]: I'm not a big fan of 'we' in a doc

**Comment by nicolasThal** [comment]
As the `kvm` solution is not used anymore, what would we do with this documentation ?

@itkg-ngilain @nbouquet @itkg-alavieille @itkg-nanne

**Comment by ngilain** [comment]
Maybe just burn it :)

**Comment by ngilain** [comment]
@alavieille , @itkg-nanne , @betd-rcarles : I close this one as it is very old, let me know if you want to reopen it

**Pull request closed without merging**